### ISSUE TITLE
Error with XSLT path expressions

### ISSUE THREAD

**Issue by emarsden** [ballsteve/xrust#96]
An XSL template with a trivial Path expression
```
     <xsl:template match='/Example' />`
```

 fails to compile in `xslt::from_document` (using the example code in the xrust::xslt documentation) with 

```
extra characters found: ""Example""
```

though my reading of the compliance section of the documentation made me think this might work.

**Comment by ballsteve** [comment]
Thanks for the bug report, Eric. I'll look into this ASAP.

You are right, this is a fairly simple pattern and should work. I'll start by adding a test for this match pattern.

**Comment by ballsteve** [comment]
Oops... rookie error ;-)

I implemented abbreviated syntax for XPath expressions, but not for XPath patterns. I'm going to work on that now on incorporate the result into the next release (long with Daniel's other changes).

**Comment by ballsteve** [comment]
A PR has been submitted with the fix.

**Comment by Devasta** [comment]
Merged to Dev now.

**Comment by ballsteve** [comment]
Fixed in release 1.1
